python/llm/src/bigdl/llm/transformers/model.py

```diff
@@ -29,6 +29,17 @@ class _BaseAutoModelClass:
     def from_pretrained(cls,
                         *args,
                         **kwargs):
+        """
+        Load a model from a directory or the HF Hub. Use load_in_4bit or load_in_low_bit parameter
+        the weight of model's linears can be loaded to low-bit format, like int4, int5 and int8.
+
+        Two new arguments are added rather than huggingface's from_pretrained, listed below.
```

Comment:
Two new arguments are added to extend Hugging Face's from_pretrained method as follows: